app/models/booth.rb

```diff
@@ -25,10 +25,15 @@ class Booth < ApplicationRecord
             :state,
             :responsibles,
             :conference_id,
+            :email,
```

Comment:
Maybe we do not want to make this mandatory? Let me know your thoughts!

Reply:
@differentreality Yes, it makes sense maybe the booth organizers would prefer the communication to happen through a public channel only like twitter etc..